assets/queries/ansible/aws/http_port_open/query.rego

```diff
@@ -0,0 +1,113 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    fromPort := task["amazon.aws.ec2_group"].rules[index].from_port
+    toPort := task["amazon.aws.ec2_group"].rules[index].to_port
+    cidr := task["amazon.aws.ec2_group"].rules[index].cidr_ip
+    cidr == "0.0.0.0/0"
+  	portNumber := 80
+  	fromPort <= portNumber
+  	toPort >= portNumber
+
+
+	result := {
+                "documentId": 		  input.document[i].id,
+                "searchKey": 	      sprintf("name={{%s}}.{{amazon.aws.ec2_group}}.rules", [task.name]),
+                "issueType":		    "IncorrectValue",
+                "keyExpectedValue": sprintf("name={{%s}}.{{amazon.aws.ec2_group}}.rules doesn't open the remote desktop port (%s)", [task.name, portNumber]),
+                "keyActualValue": 	sprintf("name={{%s}}.{{amazon.aws.ec2_group}}.rules opens the remote desktop port (%s)", [task.name, portNumber]),
+              }
+}
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    ports := task["amazon.aws.ec2_group"].rules[index].ports
+    cidr := task["amazon.aws.ec2_group"].rules[index].cidr_ip
+    cidr == "0.0.0.0/0"
+  	portNumber := 80
+  	ports == portNumber
+
+
+	result := {
+                "documentId": 		  input.document[i].id,
+                "searchKey": 	      sprintf("name={{%s}}.{{amazon.aws.ec2_group}}.rules.ports", [task.name]),
+                "issueType":		    "IncorrectValue",
+                "keyExpectedValue": sprintf("name={{%s}}.{{amazon.aws.ec2_group}}.rules.ports doesn't open the remote desktop port (%s)", [task.name, portNumber]),
+                "keyActualValue": 	sprintf("name={{%s}}.{{amazon.aws.ec2_group}}.rules.ports opens the remote desktop port (%s)", [task.name, portNumber]),
+              }
+}
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    ports := task["amazon.aws.ec2_group"].rules[index].ports
```

Comment:
In this resource's documentation, there is no mention of the attribute `ports` in rules. Are you sure this exists?